bean/internal/driver/template/subscriptions.html

```diff
@@ -30,9 +24,20 @@ <h5>
       {{ else }} {{ .Label }} {{ end }}
     </h5>
     <p>{{ .Amount }} &nbsp;·&nbsp; {{ .Frequency }}</p>
+    <p>
+      Using <a href="/cards?id={{ .PaymentMethodID }}">
+        {{ if .PaymentMethodLabel }}
+        {{ .PaymentMethodLabel }}
+        {{ else }}
+        {{ .PaymentMethodLast4 }} &nbsp;·&nbsp; {{ .PaymentMethodExpMonth }} / {{ .PaymentMethodExpYear }}
+        {{ end }}
+      </a>
+    </p>
     <p class="actionbar">
       <label for="delete-sub-{{ .ID }}">Delete</label>
     </p>
+    <br /><br />
+    <hr />
```

Comment:
Switch this to be above instead of below

We will need to use the similar logic for Payment Methods
There it has to be above so we don't check if the item is the last one or not

Reply:
375c98097d75f9358bf04d684a9f226bef729e98